src/Arcus.Security.Providers.KeyPerFile/SecretStoreBuilderExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.Extensions.Configuration.KeyPerFile;
+using Microsoft.Extensions.FileProviders;
+using Microsoft.Extensions.Hosting;
+
+namespace Arcus.Security.Providers.KeyPerFile
+{
+    public static class SecretStoreBuilderExtensions
```

Comment:
```suggestion
    /// <summary>
    /// Extensions on the <see cref="SecretStoreBuilder" /> related to Docker secrets.
    /// </summary>
    public static class SecretStoreBuilderExtensions
```

Public types documented, maybe.

Reply:
Done.